content/docs/command-reference/add.md

```diff
@@ -70,6 +72,21 @@ large files. DVC also supports other link types for use on file systems without
 `reflink` support, but they have to be specified manually. Refer to the
 `cache.type` config option in `dvc config cache` for more information.
 
+### Transferring data directly to the remote
+
+Giving `--to-remote` option would change the behavior described above. Instead
```

Comment:
Would be great to start with some motivation, a problem it solves. Like - we'd like to add data from a cloud, but with regular DVC flow it would require downloading the whole dataset first to the machine first - It can be time consuming, we might not have enough space, etc. Let's try to describe a problem first to the best of knowledge, we'll help to iterate on it.